design/MonitoringModeAPI.md

```diff
@@ -2960,6 +2960,361 @@ Returns the recommendation at a particular timestamp if it exists
+                              "memory": {
+                                "amount": 1.1090952192E9,
+                                "format": "bytes"
+                              }
+                            },
+                            "requests": {
+                              "memory": {
+                                "amount": 1.1090952192E9,
+                                "format": "bytes"
+                              }
+                            }
+                          },
+                          "notifications": {
+                            "323002": {
+                              "type": "notice",
+                              "message": "CPU usage is zero, No CPU Recommendations can be generated",
+                              "code": 323002
+                            }
+                          }
+                        }
+                      },
+                      "plots": {
+                        "datapoints": 15,
+                        "plots_data": {
+                          "2024-07-26T10:07:09.000Z": {},
+                          "2024-07-27T10:07:09.000Z": {},
+                          "2024-07-28T10:07:09.000Z": {},
+                          "2024-07-29T10:07:09.000Z": {},
+                          "2024-07-30T10:07:09.000Z": {},
+                          "2024-07-31T10:07:09.000Z": {
+                            "cpuUsage": {
+                              "min": 0.0,
+                              "q1": 0.0,
+                              "median": 0.0,
+                              "q3": 1.1810099826666658,
+                              "max": 1.2382265636666658,
+                              "format": "cores"
+                            },
+                            "memoryUsage": {
+                              "min": 0.0,
+                              "q1": 0.0,
+                              "median": 0.0,
+                              "q3": 9.23123712E8,
+                              "max": 9.24246016E8,
+                              "format": "bytes"
+                            }
+                          },
+                          "2024-08-01T10:07:09.000Z": {
+                            "cpuUsage": {
+                              "min": 0.0,
+                              "q1": 0.0,
+                              "median": 0.0,
+                              "q3": 0.0,
+                              "max": 0.0,
+                              "format": ""
+                            },
+                            "memoryUsage": {
+                              "min": 0.0,
+                              "q1": 0.0,
+                              "median": 0.0,
+                              "q3": 0.0,
+                              "max": 3.058688E8,
+                              "format": "bytes"
+                            }
+                          },
+                          "2024-08-02T10:07:09.000Z": {
```

Comment:
Format is missing here? Is box plots applicable for GPU, then we should have GPU usage also isn't it?